### PR TITLE
Juliana home design

### PR DESCRIPTION
**Pull request by julianaLeano**
As the organization, I want guests to be able to view a visually pleasing home page so that they are more inclined to join our organization.

Acceptance Criteria:

- Is mobile and PC friendly
- Styling should be evident on each page
- color scheme consistent across pages

Definition of Done:

- Revied by the product owner
- 100% code coverage
- Satisfies acceptance criteria
- Deployed on website

### CODE REVIEW

**Review by MengtingCao — CHANGES_REQUESTED** [review]
“What are some components of MCH?” -typo

**Review by MengtingCao — CHANGES_REQUESTED** [review]
Heart Image not showing up for me

**Comment by jswood23** [comment]
Part of my PR does some basic formatting of the home page. If mine gets merged first, just overwrite whatever I made but make sure to use the correct container and box imports.

File: app/assets/stylesheets/home.css [hunk 1]
```diff
@@ -1,14 +1,171 @@
 @import url("./components/containers.css");
 @import url("./components/fancy_headers.css");
 
+@import url('https://fonts.googleapis.com/css2?family=Roboto&family=Sansita&display=swap');
+
 .container {
   display: flex;
   align-items: center;
-  justify-content: center;
 }
```
Comment (jswood23) [inline app/assets/stylesheets/home.css:9]: Would you please remove the `.container` code here? This is already in the `.container` portion in `containers.css`.

**Review by jswood23 — CHANGES_REQUESTED** [review]
Looks good for the most part, just one small change needed.

**Comment by jswood23** [comment]
Could you please use a sizing measurement other than `vw` or set a maximum size? The text goes outside of the box on my screen since I'm using a wide monitor. You can replicate this by setting your screen width to 3000 in chrome inspect.
Screenshot 2022-11-12 225637

**Review by jswood23 — APPROVED** [review]
Great work! Everything looks good

**Review by MengtingCao — APPROVED** [review]
Looks good

**Pull request merged**